react/apollo/index.js

```diff
@@ -66,11 +67,20 @@ export const initApolloClient = ({
         __typename: 'SerializedMe',
         ...serializedMe,
       },
+      sharify: {
+        __typename: 'Sharify',
+      },
     },
     resolvers: {
       Cookies: {
         get: (_obj, args) => cookies[args.name] || null,
       },
+      Sharify: {
+        get: (_obj, args) => {
+          const value = sharifyData[args.name];
+          return value === undefined ? null : value;
```

Comment:
Probably should have just bit the bullet and did this earlier. Now we just access sharify through a client-side query.

Admittedly a bit weird but basically the issue is that importing it doesn't give you access to any data you might have set in the request locals. You, @broskoski, happened to just pull from an earlier SHA, hence my confusion.

Reply:
ah got it

Reply:
and yes, good idea